### PR TITLE
326 flavour to leader profile

### LINKED ISSUE #326: Adding block option to Leader profile block
Content guidelines:

- Make sure Author crops images to square to prevent the image getting pinched in the flyout
- Text should not be longer than one line - should be restricted in content guidelines 
-  Download button will truncate with ellipsis if longer than allotted space

AC:

1. Create a Block option for the existing block ( Leader Profiles) so it displays 2 columns and uses the full width of the main grid
2.  Text under the image should be in black size 26px
3. Block should have the option to have the under image text convert to a button 
4. Able to add an ordered list
5. Add the option to add a button in the flyout ([Block 5 - 8 Images mosaic with title tag*](https://projects.netcentric.biz/wiki/pages/viewpage.action?pageId=430092249))
6. The whole block (image and text) is clickable and opens the flyout

Specs:
[Block 1- Leader Profiles new Block option](https://projects.netcentric.biz/wiki/display/NCWEB/Block+1-+Leader+Profiles+new+Block+option?src=contextnavpagetreemode)
https://projects.netcentric.biz/wiki/pages/viewpage.action?pageId=430092249

Example: 
Screenshot 2023-04-24 at 13 14 06

### PR DESCRIPTION
**Pull request by sonaldhekale**
Fix #326 

Test URLs:
- Before: https://main--cn-website--netcentric.hlx.page/who-we-are
- After: https://326-flavour-to-leader-profile--cn-website--netcentric.hlx.page/drafts/sdhekale/leaderprofiledemo

### CODE REVIEW

File: blocks/leader-profiles/leader-profiles.js [hunk 1]
```diff
@@ -78,6 +78,15 @@ export default function decorate(block) {
           icon.closest('a').target = '_blank';
           icon.closest('ul').classList.add('leader-profile-social-icons');
         });
+        if (block.matches('.block-option')) {
```
Comment (nc-andreashaller) [inline blocks/leader-profiles/leader-profiles.js:81]: I think there is a misunderstanding, the name of the block option should not be "block option", that's the name of the feature itself, please clarify with @DaphneKoren what the name should be

Comment (DaphneKoren) [reply blocks/leader-profiles/leader-profiles.js:81]: Hi Yes, I think the name  "ION" should do for this block option.

**Review by mihai-buricea-net — APPROVED** [review]

**Review by nc-andreashaller — APPROVED** [review]

**Comment by alexnerdrumhansen** [comment]
Looks good, although it looks like the option to add a button is still missing.

**Comment by sonaldhekale** [comment]
> Looks good, although it looks like the option to add a button under the image is still missing.



> Looks good, although it looks like the option to add a button under the image is still missing.

i will work on this and let you know.Thank you

**Pull request merged**